### DISCUSSION TITLE
Difference between Metadata and XMP Metadata

### DISCUSSION THREAD

**Discussion by Piloudev** [py-pdf/pypdf#2212] [Q&A]
I'm asking a question to understand more things about PDF, maybe someone can help me.

When I'm looking this library and PDF in general, there is Metadata but there is also XMP Metadata.
Can someone explain me the difference between these two things ?

Thanks!

**Comment by pubpub-zz** [comment]
in PDF 1.7 spec, you can find
![image](https://github.com/py-pdf/pypdf/assets/4083478/82cbfda5-a1dc-4a4f-aeeb-081725fb07bc)

The "/metadata" can be found on individual objects (images,...) which can not be provided with the "/info" (only allowed)
at document level, "/metadata" may offer some common flags and may offer possibility to store them in multiple languages.
